Formula/libsvg.rb

```diff
@@ -23,14 +23,56 @@ class Libsvg < Formula
     sha256 cellar: :any, mavericks:   "a6de74ce690bcc7dffd353139182dc0d896250cdca652c315356349f7e78729e"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "jpeg"
   depends_on "libpng"
 
   uses_from_macos "libxml2"
 
+  # Allow building on M1 Macs. libsvg uses a very old configuration for
+  # autotools that appears now to be deprecated (Cairo itself got rid of these
+  # macros in 2008). This patch adapts the fix Cairo made to unblock
+  # installation on modern computers. It is unlikely that this patch will be
+  # accepted upstream, since as far as I understand it libsvg is in
+  # less-than-maintainence mode.
+  # (https://cgit.freedesktop.org/cairo/commit/?id=afdf3917ee86a7d8ae17f556db96478682674a76)
```

Comment:
```suggestion
  # Allow building on M1 Macs. This patch is adapted from
  # https://cgit.freedesktop.org/cairo/commit/?id=afdf3917ee86a7d8ae17f556db96478682674a76
```
Let's cut this comment down a little.

When you make this change, please squash your commits together and use the message
```
libsvg: fix for ARM/M1 Macs
```
Feel free to tweak the wording as long as you start the message with `libsvg`.

Thanks again.